src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsTrackingOptInRequestsMobileWiki.java

```diff
@@ -295,4 +296,59 @@ public void adsTrackingPixelSentAuthPageOutsideUE(List<String> urlPatterns) {
 
     modal.verifyTrackingRequestsSend(urlPatterns, networkTrafficInterceptor);
   }
+
+  @NetworkTrafficDump(useMITM = true)
+  @Execute(trackingOptIn = false)
+  @Test(dataProviderClass = TrackingOptInDataProvider.class, groups = "AdsTrackingPixelsMobileWiki", dataProvider = "adsTrackingPixelsOnConsecutivePages")
+  public void adsTrackingPixelsOnConsecutivePagesInEU(List<String> urlPatterns, String[] articles) {
+    networkTrafficInterceptor.startIntercepting();
+    TrackingOptInPage modal = new TrackingOptInPage();
+    modal.acceptOptInModal(driver, POLAND, ADS_ARTICLE1_PAGE);
+    MobileAdsBaseObject ads = new MobileAdsBaseObject();
+
+    modal.verifyTrackingRequestsSend(urlPatterns, networkTrafficInterceptor);
+
+    // Check tracking pixels on consecutive page views
+    for (String linkName : articles) {
+//      ads.waitForSlotExpanded(driver.findElement(By.cssSelector("#MOBILE_TOP_LEADERBOARD")));
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3ACommentedOutCodeLine)